src/engines/html5/html5.js

```diff
@@ -793,7 +835,17 @@ export default class Html5 extends FakeEventTarget implements IEngine {
       if (window.VTTCue && cue instanceof window.VTTCue) {
         activeCues.push(cue);
       } else if (window.TextTrackCue && cue instanceof window.TextTrackCue) {
-        activeCues.push(new Cue(cue.startTime, cue.endTime, cue.text));
+        try {
+          activeCues.push(new Cue(cue.startTime, cue.endTime, cue.text));
+        } catch (e) {
+          const message = PlayerError.createError({
+            severity: PlayerError.Severity.RECOVERABLE,
+            category: PlayerError.CATEGORY.TEXT,
+            code: PlayerError.Code.VTT_CAPTIONS_ISSUE,
```

Comment:
Name is way to vague.
You have two operations here:
1. Create VTTCue which can fail `new Cue`, as you know we are throwing errors from Cue class
2. push to active cues - no real reason to worry it will crash

A more suitable name would be UNABLE_TO_CREATE_CUE

Reply:
defintly